docs/whatsnew/2.0.rst

```diff
@@ -0,0 +1,200 @@
+SunPy now provides differential-rotation functionality integrated directly into the `coordinate framework <sunpy.coordinates>` using the `~sunpy.coordinates.metaframes.RotatedSunFrame` class.
+Here are examples of using this class:
+
+.. minigallery:: sunpy.coordinates.RotatedSunFrame
+
+
+A detailed write-up of how to use `~sunpy.coordinates.metaframes.RotatedSunFrame` can be found :ref:`here <sunpy-coordinates-rotatedsunframe>`.
+
+Changes to Carrington coordinates
+=================================
+
+We have refined our approach for heliographic Carrington coordinates to best support high-resolution imagery of the Sun, including from observatories that are at distances from the Sun that is significantly different from 1 AU (e.g., `Solar Orbiter <https://en.wikipedia.org/wiki/Solar_Orbiter>`__).
+Our `~sunpy.coordinates.frames.HeliographicCarrington` coordinate frame is now expressly intended for the co-alignment of images of the Sun's surface from different observatories.
+`~sunpy.coordinates.frames.HeliographicCarrington` now requires the specification of the observer location (Earth or otherwise) because the light travel time between the Sun and the observer is accounted for.
+SunPy output now matches the calculations by `JPL Horizons <https://ssd.jpl.nasa.gov/?horizons>`__ and `SPICE <https://naif.jpl.nasa.gov/naif/>`__.
+There may be small differences compared to Carrington coordinates computed by groups that do not use modern parameter values or the same assumptions for the methodology.
+
+Importantly, the Carrington longitude that is now calculated (including using :func:`sunpy.coordinates.sun.L0`) will not match earlier versions of SunPy.
+A detailed write-up of the calculation approach and comparisons to other resources can be found :ref:`here <sunpy-coordinates-carrington>`.
+
+Download behind proxies
+=======================
+
+With the release of parfive 1.1, sunpy has been patched now to able to pass proxy information to `aiohttp.ClientSession`.
+
+* Proxy URL is read from the environment variables `HTTP_PROXY` or `HTTPS_PROXY`.
+* Proxy Authentication `proxy_auth` should be passed as a `aiohttp.BasicAuth` object, explicitly by the user.
+* Proxy Headers `proxy_headers` should be passed as `dict` object, explicitly by the user.
+
+For example if you use a bash terminal:
+
+.. code-block:: bash
+
+    $ HTTP_PROXY=http://user:password@proxyserver.com:3128
+    $ HTTPS_PROXY=https://user:password@proxyserver.com:3128
+    $ export HTTP_PROXY
+    $ export HTTPS_PROXY
+
+these will be used to enable downloads through a proxy.
+
+Citation update
+===============
+
+A paper discussing sunpy 1.0 was accepted in The Astrophysical Journal and you can find the bibtex for it by running::
+
+    >>> import sunpy
+    >>> sunpy.__citation__
+
+or `accessing the website directly <https://iopscience.iop.org/article/10.3847/1538-4357/ab4f7a>`__.
+
+aiaprep is now deprecated
```

Comment:
now I think this is too low down the page :see_no_evil:

Reply:
Yeah I'd agree. It is a small change, but something that people obviously will want to know about